src/common/constants.ts

```diff
@@ -20,4 +20,4 @@ export const MAX_NUM_DAYS_OF_TRENDING_ARTICLES = 30;
 export const MAX_NUM_DAYS_OF_FEATURED_MAGAZINES = 30;
 export const MAX_NUM_OF_TRENDING_FLYERS = 10;
 export const UPLOAD_BUCKET = 'volume';
-export const UPLOAD_SERVICE_URL = 'https://upload.cornellappdev.com/upload/';
+export const UPLOAD_SERVICE_URL = 'https://upload.cornellappdev.com';
```

Comment:
This needs to be removed form the git history as well in accordance with #102, you should try to git reset your commits here and force a rebase onto your other branch once you've removed the sensitive information there. Will come back to this and do a full review once that's done.